### PR TITLE
Adding CDN to docs

### PR DESCRIPTION
**Pull request by rogerfranklin2**
Adding CDN information, still more work to do on the CDN side but it is useable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,32 @@ First start browserstack supplying the browserstack-key
 
 Next, run `npm run test:feature-bs -- --bskey=<PUT-THE-BROWSERSTACK-KEY-HERE>`, also supplying the same broserstack-key.
 
+
+## CDN
+
+It's recommended to use the NOW TV CDN version of react. 
+
+The convention for libraries ```/ajax/libs/<library-name>/<version>```
```
Comment (peter-mouland) [inline README.md]: probably don't need `ajax/`

**Comment by peter-mouland** [comment]
Seeing that the react-dom is only 1kb, i think we should bundle these 2 react files into a `react.bundle.js` and document what is in here to save a http request.

see https://github.com/sky-uk/nowtv-react-toolkit/issues/29#issuecomment-180051358

File: README.md [hunk 2]
```diff
@@ -110,6 +110,32 @@ First start browserstack supplying the browserstack-key
...
+- [libs/react/0.14.6/react-dom.min.js](http://web.static.nowtv.com/libs/react/0.14.6/react-dom.min.js)
+
+###
+
+### Webpack
+
+When using weback you can tell it to use the external React by adding the following to your config
+```javascript 
+externals: {
+  "react": "React"
```
Comment (peter-mouland) [inline README.md]: worth putting react-dom in here too
```

  externals: {
    "react": "React",
    "react-dom": "ReactDOM"
  },
```

**Comment by nowarkdev** [comment]
@ariley90 @malcs-sky No issues with this :+1:

**Comment by rogerfranklin2** [comment]
Ill merge this in and we can talk about the bundled version at the next toolkit catchup, as the only benefit of this is if everyone uses the same stuff

**Comment by peter-mouland** [comment]
fair enough.

**Pull request merged**